src/AmbientSounds/ViewModels/SoundViewModel.cs

```diff
@@ -161,44 +156,13 @@ public bool PlusBadgeVisible
         /// </summary>
         public string? ImagePath => _sound.IsMix ? _sound.ImagePaths[0] : _sound.ImagePath;
 
-        /// <summary>
-        /// Command for deleting this sound.
-        /// </summary>
-        public IRelayCommand DeleteCommand { get; }
-
-        public IAsyncRelayCommand RenameCommand { get; }
-
-        public IAsyncRelayCommand PlayCommand { get; }
-
         /// <summary>
         /// Returns true if the sound is currently playing.
         /// </summary>
         public bool IsCurrentlyPlaying => string.IsNullOrWhiteSpace(_playerService.CurrentMixId)
             ? _playerService.IsSoundPlaying(_sound.Id)
             : _soundMixService.IsMixPlaying(_sound.Id);
 
-        public bool DownloadActive
-        {
-            get => _downloadActive;
-            set => SetProperty(ref _downloadActive, value);
-        }
-
-        public double DownloadProgressValue
-        {
-            get => _downloadProgressValue;
-            set => SetProperty(ref _downloadProgressValue, value);
-        }
-
-        public double PresenceCount
-        {
-            get => _presenceCount;
-            set
-            {
-                SetProperty(ref _presenceCount, value);
-                OnPropertyChanged(nameof(IsPresenceVisible));
```

Comment:
This property was designed to update visibility even when the PresenceCount value didn't change. In your PR changes, does behaviour change?

Reply:
Fixed in https://github.com/jenius-apps/ambie/pull/293/commits/ae08db0206734d91404855c199193b7bfa214eb6 and made it explicit, I don't think it's good practice to notify the way it was being done earlier. Let me know what you think 🙂